swig/python/gdal-utils/osgeo_utils/rgb2pct.py

```diff
@@ -175,6 +182,13 @@ def get_parser(self, argv) -> GDALArgumentParser:
             "palette or a color file in a supported format (txt, qml, qlr).",
         )
 
+        parser.add_argument(
```

Comment:
Please use the same option name as in gdal_calc.py:
```
        parser.add_argument(
            "--creation-option",
            "--co",
            dest="creation_options",
            default=[],
            action="append",
            metavar="option",
            help="Passes a creation option to the output format driver. Multiple "
            "options may be listed. See format specific documentation for legal "
            "creation options for each format.",
        )
```

Reply:
ok, done, thanks for the suggestion